app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,14 @@
+def googleAuth
```

Comment:
This file is malformed (it needs a class definition) which is probably why the build is failing